src/domain_factory/account/depopt_claim_account.py

```diff
@@ -0,0 +1,28 @@
+from domain_factory.account.external_counterparty_account import ECPAccount
+
+
+class DepotClaimAccount(ECPAccount):
+    """ Class to generate depot claim ecp accounts. Generate method will call
```

Comment:
Very niggly, but I think it would look better if all occurrences of ecp in your comments were capitalised

Reply:
done